### ISSUE TITLE
What is wrong with powerconnect model

### ISSUE THREAD

**Issue by sajiby3k** [ytti/oxidized#1964]
I have installed oxidized 0.27.0 and because of oxidized-script. oxidized 0.26.3 version is also installed.

I have a Dell N2048 switch, where i need to send "enable" command before sending show running-config.
I have enabled this is oxidized config file, and oxidized is using api to get information from Librenms.

models:
   powerconnect:
       vars:
          enable: true

When I try show run from oxidized-script it shows the full configuration. But when oxidized try to connect and fetch the configuration, it never sends the enable command, so there is no configuration.

Can anybody explain me why oxidized-script respects enable:true from config file, but not oxidized itself ????

Output from ssh log -

As there is no enable sent, nothing works

SW>terminal datadump
                       ^
% Invalid input detected at '^' marker.

SW>terminal length 0
                       ^
% Invalid input detected at '^' marker.

SW>show version
 
Regards

**Comment by deeconwastaken** [comment]
See this post: [https://github.com/ytti/oxidized/issues/1934](url)

**Comment by mobinet-ca** [comment]
For anyone looking at this post the addition of the enable variable to the oxidized config will provide the proper functionality. The hack fix in #1934 is not recommended imho.

models:
  powerconnect:
    vars:
      enable: yourpassword

**Comment by epoxi** [comment]
I have a similar situation, but in the end is the same, I cannot obtain the configuration from Powerconnect devices (all Dell EMC Networking N1548P OS: 6.6.3.6).

The switches use credentials configured in a tacacs server rather than the local ones. Se user has the privilege level required to acquire the enable prompt. Then nothig happends... 



The log for these devices are (I picked one randomly, but are the same for all devices):

```
oxidized@srv-next-librenms-h:~/.config/oxidized/logs$ cat 172.23.251.113-ssh

HUA-PIL-SS1>enable

HUA-PIL-SS1#oxidized@srv-next-librenms-h:~/.config/oxidized/logs$ 
oxidized@srv-next-librenms-h:~/.config/oxidized/logs$ 
```

I removed the "terminal datadump" from the porwerconnect.rb as these models doesn't have the command.

Is it possible that they're reaching the timeout for the "show running-config"? Be in mind that N1548P are low-end devices (a four memeber stack takes about 20sec to show the configuration after asking for it).
